addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/studies/StudyPermissionsTable.js

```diff
@@ -109,13 +109,20 @@ class StudyPermissionsTable extends React.Component {
     const studyPermissions = this.permissionsStore.studyPermissions;
     const isEditable = studyPermissions.isStudyAdmin(this.currUser.uid) && this.study.state.canChangePermission;
 
+    if (!isEditable) {
```

Comment:
Can non-Admins still view permissions using the API?

Reply:
yep, we didn't make a change in the API as user information is required by Study Admins. Study Admins can have the researcher role. How the API should handle these tricky issues will be a part of "Project as a Boundary" feature that Jolie is working on.

Reply:
More info in GALI-900